shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -515,6 +548,9 @@ - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
     // TODO(tvolkert): provide meaningful string (e.g. "page 2 of 5")
     UIAccessibilityPostNotification(UIAccessibilityPageScrolledNotification, @"");
   }
+  if (routeChanged) {
+    UIAccessibilityPostNotification(UIAccessibilityScreenChangedNotification, previous_route_);
```

Comment:
Does native iOS actually announce the route name by default? I seem to remember that iOS usually just produces an audio clue on route changes without announcement, but I might be wrong.

Reply:
You're right it's just the chime, I was also inadvertently selecting the title when moving screens too

Reply:
There are some rare screen transitions that have announcements though. I believe, if an alert box pops up its actually announced with "Alert"??

Reply:
We could also handle this at the framework level by using `Platform` to send empty strings for the iOS case, then dispatch nil if we have the empty string so we get the "beep boop" without a phrase.

Reply:
Left with a TODO for now, we can figure it out now or come back to it later